### PR TITLE
Pam 368 misc style guide updates

### PR DESCRIPTION
**Pull request by pamgoodrich**
Addressed these new features:

1. Restrict style guide creation to workspace owners and admins
2. Import spectral file into an API project.
3. Extends cards (very basic)
4. Custom Functions.

### CODE REVIEW

**Review by mnaumanali94 — APPROVED** [review]

File: 2a.-style-guides/a1.create-style-guide.md [hunk 1]
```diff
@@ -2,16 +2,19 @@
 
 All API projects include the [Stoplight Style Guide](https://apistylebook.stoplight.io/docs/stoplight-style-guide) by default. You can use or modify the Stoplight Style Guide on any plan. On Professional and Enterprise plans, you can create your own shared style guide or import an existing Spectral ruleset as a starting point.
 
-## Shared Style Guides
-Shared style guides are a type of project that contain only style guide targets and rules. Like traditional projects, you can share the style guide with other collaborators and publish style guide documentation so others can learn about your governance program.
+## Style Guide Projects
 
-Members who can edit projects can enable style guides for those projects. See [Project Roles](../2.-workspaces/l.project-roles.md#project-roles) for more information. 
+Style guide projects only targets and rules. Like traditional projects, you can share the style guide with other collaborators and publish style guide documentation so others can learn about your governance program.
```
Comment (heitortsergent) [inline 2a.-style-guides/a1.create-style-guide.md:7]: I think the first sentence is missing something.

"Style guide projects only have targets and rules"?

File: 2a.-style-guides/a1.create-style-guide.md [hunk 2]
```diff
@@ -2,16 +2,19 @@
...
 
-### Create a Shared Style Guide
+* Only workspace owners and aministrators can create style guide projects. 
+* Style guides always have internal visibility, meaning they are only available for workspace members. 
+* Members who can edit API projects can enable style guides for those projects. See [Project Roles](../2.-workspaces/l.project-roles.md#project-roles) for more information. 
 
-To create, publish, share, and enable a style guide:
+### Create a Style Guide Project
+
+To creat a style guide project:
```
Comment (heitortsergent) [inline 2a.-style-guides/a1.create-style-guide.md:17]: create

File: 2a.-style-guides/a1.create-style-guide.md [hunk 3]
```diff
@@ -37,6 +40,19 @@ Local style guides are embedded within an API project. They cannot be shared wit
 
 You can add custom rules, targets, and functions to the default style guide. You can also override the severity of rules and disable default rules as needed.
 
+## Imported Style Guides
+
+You can import Spectral files that you've created or that have been shared with you. 
+
+When you import Spectral files into an API project, all local rules, targets, and custom functions are replaced. inherited rules, targets, and functions are not impacted.
```
Comment (heitortsergent) [inline 2a.-style-guides/a1.create-style-guide.md:47]: Inherited

File: 2a.-style-guides/h-create-custom-functions.md [hunk 4]
```diff
@@ -1,31 +1,62 @@
...
 Once you've written custom functions, add them to your style guide and use them to evaluate rules.
 
 ## Add a Custom Function
 
+![Style Guide Custom Functions](https://stoplight.io/api/v1/projects/cHJqOjI/images/a1LibPsCCqc)
+
 To create a custom function:
 
-1. Edit a style guide project or a project that has a style guide enabled.
+1. Edit a style guide project or a API project that has a style guide enabled.
```
Comment (heitortsergent) [inline 2a.-style-guides/h-create-custom-functions.md:17]: an API

File: 7.-projects/working-with-files.md [hunk 5]
```diff
@@ -22,16 +22,17 @@ Studio supports files in multiple formats:
 
 Projects use a default file directory structure. 
 
-- `/reference` - Where API descriptions documents (OpenAPI and JSON Schema) are stored
-- `models` - Where shared modles are stored
-- `/docs` - Where articles (Markdown files) are stored
-- `/assets/images` - Where images are stored
+- `/root` - Stoplight configuration files
+- `/reference` - API descriptions documents (OpenAPI and JSON Schema)
+- `models` - Shared models
```
Comment (heitortsergent) [inline 7.-projects/working-with-files.md:27]: Should this have the "/" before it like the other items?

**Pull request merged**